lib/archethic/bootstrap/sync.ex

```diff
@@ -133,20 +135,27 @@ defmodule Archethic.Bootstrap.Sync do
   @doc """
   Fetch and load the nodes list
   """
-  @spec load_node_list(list(Node.t())) :: :ok | {:error, :network_issue}
-  def load_node_list([node | rest]) do
-    case P2P.send_message(node, %ListNodes{}) do
+  @spec load_node_list() :: :ok | {:error, :network_issue}
+  def load_node_list() do
+    current_nodes = P2P.authorized_and_available_nodes()
+
+    conflict_resolver = fn results ->
+      Enum.max_by(results, fn %NodeList{nodes: nodes} -> length(nodes) end)
```

Comment:
I don't think `length(nodes)` is a good health/synch indicator. 
Maybe we should take the one with the highest `node.last_update_date` ?

Reply:
That's a good idea, but I think we still need both, the length (to have the maximum number of node) and the last update date.
We can have a reduce function that aggregate all the results and pick the last updated node.